src/cli/get-hooks.js

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env node
 console.log(JSON.stringify({
   hooks: {
-    'get-manifest': 'npm exec --package=@slack/bolt slack-cli-get-manifest',
-    start: 'npm exec --package=@slack/bolt slack-cli-start',
+    'get-manifest': 'npx -q --no-install -p @slack/bolt slack-cli-get-manifest',
```

Comment:
This was added earlier by @filmaj, but got lost in version history so I am adding it back. npm exec is not yet supported by older versions of node. `npx` works with  v12.22.12 (npm v6.14.16), node v14.19.2 (npm v6.14.17) and node v16.14.0 (npm v8.3.1). Looks like `npx` is a bit slower in general, I was seeing these commands run with noticeable lag on node v14.9.2 and upgrading to latest stable node seemed to resolve that. npx in newer versions uses npm exec under the hood anyway, probably why.  

Anyway worth noting in case people start noticing lag, that it may be due to slow `get-manifest` or `get-hooks` and that they should try upgrading their node version.